### ISSUE TITLE
node-sass moved to dependencies in v5.5.3

### ISSUE THREAD

**Issue by RSWilli** [olifolkerd/tabulator#4376]
**Describe the bug**
In https://github.com/olifolkerd/tabulator/commit/538427780d31936ac02281f9dcec3b37841e1af2 the node-sass dependency was moved from dev dependencies to dependencies. I can only assume that this was a mishap while upgrading from v7 to v9.

Node-sass is deprecated and triggers a node-gyp compilation on my machine and in our CI pipeline, which crashes or times out since it is not setup correctly.

**Tabulator Info**
- v5.5.3

**Working Example**
v5.5.2 does not have this dependency https://www.npmjs.com/package/tabulator-tables/v/5.5.2?activeTab=dependencies

**To Reproduce**
`npm i tabulator-tables@5.5.3`

**Expected behavior**
Tabulator does not need node-sass in it's runtime, so I'd expect this to be a dev dependency

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Arch Linux

**Comment by danieleperilli** [comment]
I confirm that this caused me several problems, as I didn't realize right away that it was caused by Tabulator.
With version 5.5.3, I get this error when run npm install/update:

```terminal
npm ERR! code 1
npm ERR! path /src/node_modules/node-sass
npm ERR! command failed
npm ERR! command sh -c node scripts/build.js
npm ERR! Binary found at /src/node_modules/node-sass/vendor/darwin-arm64-115/binding.node
npm ERR! Testing binary
npm ERR! Binary has a problem: Error: dlopen(/src/node_modules/node-sass/vendor/darwin-arm64-115/binding.node, 0x0001): tried: '/src/node_modules/node-sass/vendor/darwin-arm64-115/binding.node' (not a mach-o file), '/System/Volumes/Preboot/Cryptexes/OS/src/node_modules/node-sass/vendor/darwin-arm64-115/binding.node' (no such file), '/src/node_modules/node-sass/vendor/darwin-arm64-115/binding.node' (not a mach-o file)
```

**Comment by olifolkerd** [comment]
I have just released 5.5.4 to fix this, sorry for the regression there

Cheers

Oli :)
